src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -1061,4 +1063,18 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
       window.scrollTo({ top: y, behavior: 'smooth' });
     }, 0);
   }
+
+  scrollUp(): void {
+    window.scrollTo({ top: 0, behavior: 'smooth' });
+  }
+  scroll(): void {
```

Comment:
I believe this function is not needed anymore since we've removed `scrollToTopButton`?

Reply:
I am still calling `scrollUp()` with the link, I use the same function just not in the button format

Reply:
I can see how `scrollUp()` is being used, but I was referring to `scroll()` which I can see is adding an event listener on scroll, but it doesn't seem to be changing the className of any component since `btn-to-the-top` has been removed